### PR TITLE
Feature : playlist

### PR DESCRIPTION
**Pull request by haarsh24**

### CODE REVIEW

**Review by yashpatel1797 — APPROVED** [review]
code looks clean
file structure is good
design looks consistent

File: src/components/playListInfoCard/playListInfoCard.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    const { playlistId } = useParams();
+
+    const playlistDeleteHandler = () => {
+        deleteButton(playListDispatch, playlistId,token);
+        playlistId && navigate("/playlist");
+    }
+    return (
+        <>
+            <div className="">
+                <div className="">
```
Comment (xii-vi) [inline src/components/playListInfoCard/playListInfoCard.jsx:19]: remove className if it is empty

File: src/components/playListInfoCard/playListInfoCard.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                    <img
+                        src={`https://i.ytimg.com/vi/${image_id}/hq720.jpg`}
+                        alt=""
```
Comment (xii-vi) [inline src/components/playListInfoCard/playListInfoCard.jsx:22]: _give alt value_

File: src/components/playlistModal/playlistModal.jsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+import { isVideoInArray } from "../../utilities/helperFunctions/isVideoInArray";
+import { usePlayList} from "../../context/playlistContext";
+import { useAuth } from "../../context/authContext";
+import { createPlayList } from "../../utilities/apiCalls/playlistModalApiCalls";
+import {isVideoAlreadyInPlaylist} from "../../utilities/helperFunctions/isVideoAlreadyInPlaylist";
+import {addVideoInPlaylist, removeVideoFromPlaylist} from "../../utilities/apiCalls/playlistApiCalls";
+
+const PlaylistModal = ({ video }) => {
+    const [playlistInput, setPlaylistInput] = useState(false)
+    console.log(video)
```
Comment (xii-vi) [inline src/components/playlistModal/playlistModal.jsx:11]: remove console.log

File: src/components/playlistModal/playlistModal.jsx [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    const { setShowModal, playlists, playListTitle, playListDispatch, playListTitleDispatch } = usePlayList();
+    const { authState: { token } } = useAuth();
+    console.log(playlists);
+    console.log(token);
```
Comment (xii-vi) [inline src/components/playlistModal/playlistModal.jsx:15]: here too remove it

File: src/components/plylistCard/playlistCard.jsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+const image_id = videos.length > 0 ? `${videos[0]._id}` : "Sxjo5MySQFI"
+
+const playlistDeleteHandler = () => {
+    deletePlaylist(playListDispatch, _id,token);
+}
+    return (
+    <>
+    <div className=" flex-column p1">
+        <Link to={`/playlist/${_id}`} >
+            <div className="">
```
Comment (xii-vi) [inline src/components/plylistCard/playlistCard.jsx:19]: here too remove the empty className

**Review by xii-vi — APPROVED** [review]
_Minor suggested change in comments.
Please go through them once before merging._

**Pull request merged**